StarWarsExplorer.js

```diff
@@ -0,0 +1,490 @@
+        console.clear();
+        goWest();
+    } else if (input === 'ladder') {
+        console.clear();
+        useLadder();
+    } else if (input === 'exit') {
+        console.clear();
+        exitShip();
+    } else if (input === 'use') {
+        // console.clear();
+        useItem();
+    } else if (input === 'map') {
+        console.clear();
+        showMap();
+        console.log(`You are currently in the ${mapBuild.map[initialX][initialY]}`);
+    } else if (input === 'menu') {
+        console.clear();
+        menuOptions[1] = "Resume Game";
+        menuOptions[2] = "Save Game(Work in Progress)";
+        menuOptions[3] = "Load Game(Work in Progress)";
+        menu();
+    } else if (input === 'help') {
+        console.clear();
+        showHelp();
+    } else if (input === 'quit') {
+        console.clear();
+        quitGame();
+    }
+}
+const goNorth = () => {
+    if ((mapBuild.map[initialX][initialY] === 'Bridge') || (mapBuild.map[initialX][initialY] === 'Conference Room') || (mapBuild.map[initialX][initialY] === 'Captain\'s Quarters')){
+        if (mapBuild.map[initialX][initialY] === 'Conference Room'){
+            x;
+            y--;
+            isOnUpperLv(x,y);
+        } else if (mapBuild.map[initialX][initialY] === 'Captain\'s Quarters') {
+            x;
+            y--;
+            isOnUpperLv(x,y);
+        } else {
+            cantMove();
+        }
+    }
+    /*
+    // if ((x === 0 && y === 0) || (x === 0 && y === 1) || (x === 0 && y === 2)) {
+    //     if (x === 0 && y === 1) {
+    //         x;
+    //         y--;
+    //         isOnUpperLv(x,y);
+    //     } else if (x === 0 && y === 2) {
+    //         x;
+    //         y--;
+    //         isOnUpperLv(x,y);
+    //     } else {
+    //         cantMove();
+    //     }
+    }*/ else {
+        if ((x === 1 && y === 0) || (x === 1 && y === 1) || (x === 1 && y === 2)) {
+            cantMove();
+        } else {
+            if (x === 2 && y === 0) {
+                // if you are in Crew Quarters
+                // you go north to Med Bay
+                x--;
+                isOnLowerLv(x,y);
+            } else if (x === 3 && y === 0) {
+                // if you are in Crew Quaters
+                // you go north to Escape Pod
+                x--;
+                isOnLowerLv(x,y);
+            } else if (x === 2 && y === 1) {
+                // if you are in Main Hold
+                // you go north to Engine Room
+                x--;
+                isOnLowerLv(x,y);
+            } else if (x === 3 && y === 1) {
+                // if you are in Ramp
+                // you go north to Main Hold
+                x--;
+                isOnLowerLv(x,y);
+            } else if (x === 2 && y === 2) {
+                // if you are in Cargo Elevator
+                // you go north to Cargo Hold
+                x--;
+                isOnLowerLv(x,y);
+            } else if (x === 3 && y === 2) {
```

Comment:
These blocks all do the same thing. Think about ways to fix. Maybe a function that checks these coords? Using ||